release/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id "com.palantir.docker" version "0.25.0"
+}
+
+dependencies {
+    compile project(':situp-core')
+}
+
+task endToEndTests {
+    //dependsOn(':situp-api:test') TODO add E2E and enable when ready
+}
+
+task benchmarkTests {
+    //dependsOn(':situp-api:test') TODO add benchmark test and enable
+}
+
+task buildCore {
+    dependsOn ':situp-core:build'
+}
+
+task dockerPrerequisites {
+    dependsOn 'buildCore'
+    dependsOn 'endToEndTests'
+    dependsOn 'benchmarkTests'
+}
+
+docker {
+    name "odfe-pipes/${project.rootProject.name}:${project.version}"
+    tag "${project.name}", "${project.version}" //update tag when ready to push
```

Comment:
QUES: are the tags going to be updated inline or in a separate config file?

Reply:
after we decide the tag name pattern, we can update it either ways - would prefer to read it from a global file which has details about the release and appropriate release notes